Readme.md

```diff
@@ -17,6 +17,8 @@ Published package: https://www.nuget.org/packages/GivePenny.GherkinSpec.TestAdap
 
 ## Getting started
 
+**If you're upgrading to a new major version (e.g. 1.x to 2.0) please read [the changelog]()**
```

Comment:
Should this be `[the changelog](Changelog.md)`?

Reply:
Thanks, fixed